packages/next/build/webpack/loaders/next-serverless-loader.ts

```diff
@@ -127,6 +127,10 @@ const nextServerlessLoader: loader.Loader = function() {
           .replace(/\\.json$/, '')
       }
       const parsedUrl = parse(req.url, true)
+
+      const { _nextOrigPathname } = parsedUrl.query
```

Comment:
This doesn't seem right. We wouldn't want the original URL, that'd be the `req.url`.

Did you mean `_nextRewrotePathname`?

I'm assuming we're not overriding `req.url` (and hope we're not).

Reply:
Second, this approach feels weird. How will this work on Now itself?

Reply:
This is only needed for `next start` since the serverless lambdas don't have their own router to process the rewrites. We pass the result that Now or `next-server` has from following the `rewrites` for the lambda to use

Reply:
Renamed to `_nextRewrotePathname` so it's more clear